script/test

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+bundle install
```

Comment:
why?

Reply:
To ensure recent (1.6.2) cocoapods version on CI. Is there a better way to do it?